components/modal/index.zh-CN.md

```diff
@@ -99,6 +99,11 @@ modal.update({
   content: '修改的内容',
 });
 
+modal.update(prevConfig => ({
```

Comment:
please add api version

Reply:
What version should I use? Is `4.7.3` ok?

Reply:
4.8.0

Reply:
Docs updated in f971a718